server/publications/collections/orders.js

```diff
@@ -105,8 +32,28 @@ Meteor.publish("Orders", function () {
  * paginated orders
  */
 
-Meteor.publish("PaginatedOrders", function (filter, limit) {
-  check(filter, Match.OptionalOrNull(String));
+ Meteor.publish("NewPaginatedOrders", function (query, options) {
```

Comment:
Including "New" in the name of almost anything is a bad idea. Because it won't be very long until it's not "New"